packages/components/src/DataList/DataList.stories.tsx

```diff
@@ -97,6 +98,7 @@ const Template: ComponentStory<typeof DataList> = args => {
         created: "Created",
       }}
       loading={loadingInitialContent}
+      totalCount={data?.allPeople.totalCount}
```

Comment:
if you move these 2 above `{...args}` you should be able to override them with the `Basic.args`